ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -8,7 +8,7 @@ Resource       ./Workbenches.resource
 ${DS_PROJECT_TITLE}=    Data Science Projects
 ${TITLE_INPUT_XP}=    xpath=//input[@id="manage-project-modal-name"]
 ${DESCR_INPUT_XP}=    xpath=//textarea[@id="manage-project-modal-description"]
-${RESOURCE_INPUT_XP}=    xpath=//input[@data-testid="resource-manage-project-modal-name"]
+${RESOURCE_INPUT_XP}=    css:[data-testid="manage-project-modal-name"]
```

Comment:
```suggestion
${RESOURCE_INPUT_XP}=    css:[data-testid="resource-manage-project-modal-name"]
```

Reply:
Also, are you sure that this change works fine? Changing from xpath to css (AFAIK equivalent for the `document.querySelector()` in the browser console) - will this work with all the current places this variable is being used now? :thinking:

Reply:
this is actually the root cause of previous comment. was pointing to the wrong selector, need to clean up

re css: this is the best way going forward according to @manosnoam , by looking for the specific data-testid only as it should be unique

Reply:
I understand the motivation; what I'm not sure is the `css:` prefix there - will it work just fine with the current places this is being used? Shouldn't we have `//` prefix there?

Disclaimer: I didn't test it and I know basically nothing about this and html and stuff around it.

Reply:
> I understand the motivation; what I'm not sure is the css: prefix there - will it work just fine with the current places this is being used? Shouldn't we have //* prefix there?

This is just SeleniumLibrary syntax for XPaths and css selectors; not really web-stuff related https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#:~:text=can%20be%20implicit.-,Default%20locator%20strategy,-By%20default%2C%20locators

Reply:
thanks for your help @jstourac & @jiridanek